job/replay_manager.go

```diff
@@ -90,6 +109,116 @@ func (m *Manager) Replay(reqInput *models.ReplayWorkerRequest) (string, error) {
 	}
 }
 
+func (m *Manager) validate(ctx context.Context, replaySpecRepo store.ReplaySpecRepository, reqInput *models.ReplayWorkerRequest) error {
+	reqReplayTree, err := prepareTree(reqInput)
+	if err != nil {
+		return err
+	}
+	reqReplayNodes := reqReplayTree.GetAllNodes()
```

Comment:
return value of this call is only getting used in if statement then why put it here?